From 4edc6a771fb478f7da8d9e82f6b5f2dd6a85c4ff Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Sun, 11 May 2014 22:15:40 -0700
Subject: [PATCH] autoconf: fix typo in description generated by AC_RUN_IFELSE

* lib/autoconf/general.m4 (_AC_RUN_IFELSE): "run" not "link"
in description.
---
 lib/autoconf/general.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/autoconf/general.m4 b/lib/autoconf/general.m4
index 86c1e73..e2e9b6a 100644
--- a/lib/autoconf/general.m4
+++ b/lib/autoconf/general.m4
@@ -2778,7 +2778,7 @@ m4_define([_AC_RUN_IFELSE_BODY],
 AC_DEFUN([_AC_RUN_IFELSE],
 [AC_REQUIRE_SHELL_FN([ac_fn_]_AC_LANG_ABBREV[_try_run],
   [AS_FUNCTION_DESCRIBE([ac_fn_]_AC_LANG_ABBREV[_try_run], [LINENO],
-    [Try to link conftest.$ac_ext, and return whether this succeeded.
+    [Try to run conftest.$ac_ext, and return whether this succeeded.
      Assumes that executables *can* be run.])],
   [$0_BODY])]dnl
 [m4_ifvaln([$1], [AC_LANG_CONFTEST([$1])])]dnl
-- 
1.9.1

